tests/monster_test_generated.h

```diff
@@ -1,3403 +0,0 @@
-// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Why is this file deleted? We have generated files in the repo to review code generator changes. Please leave it in.

Reply:
I can't use test.cpp for testing the starlark and have this file checked in.  You are asking me to do 2 conflicting things.  Which of the two is more important to you?  How you want me to resolve this?  The starlark rules now work.  I am struggling to do what you want to finish the rest of the review.

Reply:
Sorry, I have no idea what starlark does, so I didn't know there was a conflict. Having this file not be deleted is definitely the most important. Surely you must be able to generate this file somewhere else, and still build with the existing `test.cpp` ? This being a build system I'd hope you can specify an additional include dir somewhere.